activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-rest/src/test/java/org/activiti/cloud/services/query/rest/TaskEntityControllerIT.java

```diff
@@ -114,7 +114,7 @@ void setUp() {
     }
 
     @Test
-    public void findAllShouldReturnAllResultsUsingAlfrescoMetadataWhenMediaTypeIsApplicationJson() throws Exception {
+    void findAllShouldReturnAllResultsUsingAlfrescoMetadataWhenMediaTypeIsApplicationJson() throws Exception {
```

Comment:
Can you please follow the test naming convention? 🙏 

https://hyland.atlassian.net/wiki/spaces/HXP/pages/1308014107/Studio+-+Coding+Style+Guidelines#Naming-and-other-test-conventions

Reply:
Also, the public access modifier is no longer needed in JUnit 5 tests, which is great news!